.eslintrc.js

```diff
@@ -28,7 +28,6 @@ module.exports = {
       }
     ],
     'no-console': 'error',
-    eqeqeq: ['error', 'always'],
-    curly: ['error']
+    eqeqeq: ['error', 'always']
```

Comment:
Please don't remove lint rules.

Reply:
Ahh... sorry for removing the lint. I jt just felt odd writing a guard clause with curly brackets cause its normally written without them.